backend/services/rabbit-service/src/publisher.ts

```
@@ -0,0 +1,12 @@
+import {Channel} from 'amqplib';
+
+export async function sendDataToMessageBroker(routingKey: string, data:any, channel:Promise<Channel>, exchangeName:string): Promise<void> {
+    try {
+        const message = JSON.stringify(data);
+        await (await channel).assertExchange(exchangeName, 'topic', { durable: true });
+        (await channel).publish(exchangeName, routingKey, Buffer.from(message), { persistent: true });
+
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)